### PR TITLE
Templating for issues with labels

### PR DESCRIPTION
**Pull request by Theoklitos**
Added a checkbox in the template creation page, that enables it to submit issues with labels. The labels are entered in a textfield in the form of comma-separated values.

I can add some styling if you think my idea is OK.

### CODE REVIEW

File: app/services/TemplateService.js [hunk 1]
```diff
@@ -38,6 +39,7 @@ angular.module('it').factory('TemplateService', function($q, Firebase, util, _)
       var owner = cleanOutboundReference(obj.owner);
       var repo = cleanOutboundReference(obj.repo);
       var name = cleanOutboundReference(obj.name);
+      var labels = obj.labels
```
Comment (kentcdodds) [inline app/services/TemplateService.js]: I don't think this line is necessary.

File: app/new-issue/NewIssueCtrl.js [hunk 2]
```diff
@@ -4,6 +4,7 @@ angular.module('it').controller('NewIssueCtrl', function($scope, $sce, $filter,
     $scope.owner = $routeParams.owner;
     $scope.repo = $routeParams.repo;
     $scope.name = $routeParams.name;
+    $scope.labels = $routeParams.labels;
```
Comment (kentcdodds) [inline app/new-issue/NewIssueCtrl.js]: I don't like the idea of `labels` being a part of the URL. They should be more like the issue template's description. Simply meta-data about a specific template that's identified by the repo owner, repo name, and the template name.

**Comment by Theoklitos** [comment]
Also removed unnecessary lines in TemplateService and TemplateCtrl etc

**Comment by kentcdodds** [comment]
Sorry! Didn't get to this PR this morning. I'll have to try again tomorrow morning. I hope it's not too inconvenient.

**Comment by Theoklitos** [comment]
Of course not, you are doing us a service for even bothering with this label stuff :smile:

**Comment by kentcdodds** [comment]
Totally awesome. Thanks!

**Pull request merged**